packages/server/src/tests/banners/bannerDeploySchedule.ts

```diff
@@ -43,7 +43,11 @@ export const previousScheduledDate = (now: Date, dayOfWeek: number, hour: number
             ? subDays(now, 1) // scheduled for later today, so go back to previous week
             : now;
 
-    return setHours(previousDay(date, dayOfWeek), hour);
+    return set(previousDay(date, dayOfWeek), {
+        hours: hour,
+        minutes: 0,
+        seconds: 0,
+    });
```

Comment:
Okay I think I understand - so before this we were effectively redeploying the banner every second for the first hour?

Reply:
yep, every page view in the first hour would see that banner